### PR TITLE
Add new features vrrp_track_file, track_files, (no) accept, and safeguards.

### PR DESCRIPTION
**Pull request by Kariton**
As discussed in (draft) PR [1], we need to split the new features
and the general code updates. This tackles the feature additions
that are pending for a while.

Closes: #140  #176 #195 

[1]: https://github.com/evrardjp/ansible-keepalived/pull/203

### CODE REVIEW

**Comment by Kariton** [comment]
#205 fixes the 
> var-spacing: Jinja2 variables and filters should have spaces before and after.

necessary changes are made here: https://github.com/evrardjp/ansible-keepalived/pull/205/commits/f4aa7a83dbfa7079b5ae8913d560b9e7cf6483e5

**Comment by Kariton** [comment]
looks fine now.

except:
> schema: 7 is not one of ['6.1', '7.1', '7.2', 'all'] (schema[meta])

**Comment by Kariton** [comment]
forgot to push...

**Comment by evrardjp** [comment]
>     schema: 7 is not one of ['6.1', '7.1', '7.2', 'all'] (schema[meta])

will be fixed soon. Right now, you can temporarily constrain the max version of ansible-lint if you like.

File: .ansible-lint [hunk 1]
```diff
@@ -5,3 +5,6 @@
 skip_list:
   - '106' # Role name {} does not match ``^[a-z][a-z0-9_]+$`` pattern
   - 'fqcn-builtins'
+
+warn_list:  # or 'skip_list' to silence them completely
```
Comment (evrardjp) [inline .ansible-lint:9]: can you explain why you added this? 

I am rebasing your changes, so I will add this to the commit message.

Comment (Kariton) [reply .ansible-lint:8]: IMO a fail on an ``experimental`` lint feature is not feasible.

Its more a safety measure.

**Comment by evrardjp** [comment]
I have updated the PR text, so that it automatically closes issues/PR. I will update the commit messages so we can merge this.

Great work.

**Pull request merged**